web-app/src/eventHandlers/pullRequestEventHandler.ts

```diff
@@ -180,7 +180,6 @@ export class PullRequestEventHandler {
             await appGitHubService.createPullRequestComment({
               ...pullInfo,
               commit_id: pullRequest.head.sha,
-              start_line: 1,
               end_line: 1,
```

Comment:
Also fixed the error comment not showing up because the start line and end line cannot be the same. Just removed the start line completely to fix this.